bower.json

```diff
@@ -33,6 +33,7 @@
     "vaadin-themable-mixin": "vaadin/vaadin-themable-mixin#^1.3.2",
     "vaadin-text-field": "vaadin/vaadin-text-field#^2.1.1",
     "vaadin-list-box": "vaadin/vaadin-list-box#^1.1.0",
+    "vaadin-list-mixin": "vaadin/vaadin-list-mixin#^2.2.0",
```

Comment:
This change doesn't seem right to me. The list-mixin should be picked up as a transitive dependency of the list-box. So we should release 1.2.0 of the list-box first.

Reply:
Discussed in person with @tomivirkki and I agree with his reasoning stated above.